backend/routes/api.php

```diff
@@ -31,6 +31,7 @@
 	Route::post('categories/add', [CategoryController::class, 'store']);
 	Route::get('categories/questions', [CategoryController::class, 'categoryQuestions']);
 	Route::put('categories/{categoryId}/edit', [CategoryController::class, 'updateCategoryDetails']);
+	Route::delete('categories/{categoryId}/delete', [CategoryController::class, 'deleteCategory']);
 	Route::post('categories/{categoryId}/question/add', [QuestionController::class, 'addQuestionWithChoicesAndAnswer']);
```

Comment:
1. You can group them by controller
2. No need to mention `/delete` or `/delete` on routing since its already defined in the method. It will not conflict
3. Use the laravel convention on creating methods/functions/routes
https://laravel.com/docs/9.x/controllers#actions-handled-by-resource-controller

```suggestion
    Route::controller(QuestionController::class)->group(function () {
	    Route::post('categories/add', 'store');
	    Route::get('categories/questions', 'index');	
	    Route::put('categories/{categoryId}', 'update');
	Route::delete('categories/{categoryId}', 'delete');
    });
```